sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/IncrementalExecution.scala

```diff
@@ -75,14 +74,11 @@ class IncrementalExecution(
    * with the desired literal
    */
   override lazy val optimizedPlan: LogicalPlan = {
-    val random = new Random()
```

Comment:
why do we remove this central `Random` for setting the seeds?

Reply:
I saw Rand/Randn use`Utils.random`, so changed to use it consistently.